gtfs/spec/en/reference.md

```diff
@@ -178,16 +186,18 @@ File: **Required**
 
 File: **Required**
 
+Primary ID (`trip_id`)
+
 |  Field Name | Type | Presence | Description |
 |  ------ | ------ | ------ | ------ |
-|  `route_id` | ID referencing `routes.route_id` | **Required** | Identifies a route. |
-|  `service_id` | ID referencing `calendar.service_id` or `calendar_dates.service_id` | **Required** | Identifies a set of dates when service is available for one or more routes. |
-|  `trip_id` | ID | **Required** | Identifies a trip. |
+|  `route_id` | Foreign ID referencing `routes.route_id` | **Required** | Identifies a route. |
+|  `service_id` | Foreign ID referencing `calendar.service_id` or `calendar_dates.service_id` | **Required** | Identifies a set of dates when service is available for one or more routes. |
```

Comment:
This should be and/or. Because the file could contain either a normalised, normalised+exceptions, or denormalised setup.

Reply:
I left the original phrasing of the field, only adding that it was a "Foreign" ID. At least in American usage "or" is taken to mean one, the other or both (like logical OR), and it seems pretty clear here that "or" isn't intended to mean "not both".